test/e2e/framework/seclist_utils.go

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
A set of utility functions that count the number of ingress/egress port rules in the specified seclists. Used by the test check the right number of rules are mutated and all the old rules are deleted.